app/CodeFormatter/PhpCodeFormatter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DevCommunityDE\CodeFormatter\CodeFormatter;
+
+/**
+ * Class PhpCodeFormatter.
+ */
+class PhpCodeFormatter extends CodeFormatter
+{
+    /**
+     * @var array
+     */
+    protected static $supported_languages = [
+        'php',
+    ];
+
+    /**
+     * @param string $file
+     *
+     * @return string
+     */
+    protected function getShellCommand(string $file): string
+    {
+        return 'php-cs-fixer fix ' . $file;
```

Comment:
i recently rewrote the formatter-logic to use pipes instead of files

```suggestion
        return 'php-cs-fixer fix -';
```